net/microsocks/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=microsocks
+PKG_VERSION:=1.0.3
+PKG_RELEASE:=1
+PKG_SOURCE:=microsocks-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=http://ftp.barfooze.de/pub/sabotage/tarballs/
+PKG_HASH:=02449160c505ff7ea1bcb71ae781dafcfe66ab95287f3a4fca11621b74b2d7ac
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Mateusz Korniak <matkorgithubcom@ant.gliwice.pl>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/microsocks
+  SECTION:=net
+  SUBMENU:=Web Servers/Proxies
+  CATEGORY:=Network
+  TITLE:=SOCKS5 TCP/IP only proxy
+endef
+
+define Package/microsocks/description
+SOCKS5 only and TCP/IP only proxy server
```

Comment:
This description seems to be like the title. I'd say not very well described.

Reply:
There is not much available. Made description more verbose.